source/OctoVersion.Tests/ConfigurationBootstrapperFixture.cs

```diff
@@ -15,12 +15,15 @@ public class ConfigurationBootstrapperFixture : IDisposable
     {
         public void Dispose()
         {
-            foreach (DictionaryEntry environmentVariable in Environment.GetEnvironmentVariables())
-            {
-                var key = (string)environmentVariable.Key;
-                if (key.StartsWith(ConfigurationBootstrapper.EnvironmentVariablePrefix))
-                    Environment.SetEnvironmentVariable(key, "");
-            }
+            var envVars = Environment.GetEnvironmentVariables()
+                .Cast<object?>()
```

Comment:
I'm surprised that a `Cast` is required here?

Reply:
Yeah, it's not great. The interface `Environment.GetEnvironmentVariables()` is a weird one. It makes me sad.